Release 36 (#29)
JupiterOne/docs: release-notes/2019-36.md

# JupiterOne 2019.36 Release

2019-12-23

## New Features

- Lots of UI/UX updates to the **Insights app**:

>   - Added the ability to drag-n-drop an Insights board to re-arrange them in the

Reviewer: ```suggestion
  - Added the ability to drag-n-drop Insights boards to re-arrange them in the
```

`an Insights board` is singular but `them` is a plural pronoun. This change would fix that mismatch.

    menu side drawer.

  - Added the ability to clone a board. Admin users can also clone a board and
    save it as a new shared/team board.

  - Improved UX for adding and updating boards.

- Many UI/UX updates to the **Alerts app** as well:

  - Visual rule editor now supports updated rule actions, including **Webhook**
    action, **email body**, and **templates** definitions.

  - Visual editor now works when editing an existing rule. Previously editing
    was only supported via the advanced JSON rule editor.

> - [ENTERPRISE accounts only] Added the ability to **set query permissions** to a

Reviewer: ```suggestion
- [ENTERPRISE accounts only] Added the ability to **set query permissions** for a
```

>   user group, which applies a filter constrain on the data users in that group

Reviewer: ```suggestion
  user group, which applies a filter constraint on the data users in that group
```
